src/js/messages.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+import { interpolate } from './util';
+import { MESSAGES, RICH_CONTENT_MESSAGES } from './config';
+
+export function getMessage (item, { MAINTENANCE_MODE, contributor_content }) {
```

Comment:
Not going to check this. I assume you just copied it from one file to another.

Reply:
Correct.

Reply:
Tests need to be written around this but as it's not part of my changes I suggest these are handled as part of https://financialtimes.atlassian.net/browse/ACC-644